libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -184,10 +187,19 @@ - (id) initWithConfig:(SFHybridViewConfig *) viewConfig useUIWebView:(BOOL) useU
         _hybridViewConfig = (viewConfig == nil ? [SFHybridViewConfig fromDefaultConfigFile] : viewConfig);
         NSAssert(_hybridViewConfig != nil, @"_hybridViewConfig was not properly initialized. See output log for errors.");
         self.startPage = _hybridViewConfig.startPage;
-        
+
         // Setup global stores and syncs defined in static configs
         [[SalesforceHybridSDKManager sharedManager] setupGlobalStoreFromDefaultConfig];
         [[SalesforceHybridSDKManager sharedManager] setupGlobalSyncsFromDefaultConfig];
+        __weak typeof(self) weakSelf = self;
+        [SFSDKAuthConfigUtil getMyDomainAuthConfig:^(SFOAuthOrgAuthConfiguration *authConfig, NSError *error) {
+            __strong typeof(weakSelf) strongSelf = weakSelf;
+            dispatch_async(dispatch_get_main_queue(), ^{
+                if (!error) {
+                    strongSelf.authConfig = authConfig;
```

Comment:
Fetching org config once as part of view controller initialization and saving it.